dojo/tools/nikto/parser.py

```diff
@@ -62,86 +139,42 @@ def process_scandetail(self, scan, test, dupes):
             severity = "Info"  # Nikto doesn't assign severity, default to Info
 
             # Description
-            description = "\n \n".join((("Host: " + ip),
-                                    ("Description: " + item.find("description").text),
-                                    ("HTTP Method: " + item.attrib["method"]),
-                                    ))
-            mitigation = "N/A"
-            impact = "N/A"
-            references = "N/A"
-
-            dupe_key = hashlib.md5(description.encode("utf-8")).hexdigest()
+            description = "\n".join([
+                    f"**Host:** `{ip}`",
```

Comment:
I was about to say we need to be compatible with 3.6... but these were introduced in 3.6, good thing then! :-)